### DISCUSSION TITLE
Utilize commitlint to enforce commit message syntax on PRs

### DISCUSSION THREAD

**Discussion by stramel** [Buuntu/react-final-table#42] [Ideas]
Using a Github Action, like https://github.com/marketplace/actions/commit-linter, we can enforce the format of commit messages in the PR stage.

You can extend this one to start: https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-angular

**Comment by Buuntu** [comment]
I would accept a PR for this.  At this stage personally I'd want to add functionality to the library so it gets more adoption/contributions, but this GH action can't hurt!  I started a new discussion around adding features btw: https://github.com/Buuntu/react-final-table/discussions/44

**Comment by stramel** [comment]
Might be better to use https://github.com/zeke/semantic-pull-requests

If you use the "Squash and merge" option, the PR title should be the commit message and then you don't have to worry about contributors committing a specific syntax.

However, either way, with this action/app we can enforce the title, commits, or both
